free-podcasts-screencasts-en.md

```diff
@@ -103,8 +103,8 @@
 * [Podcast] - [Software Engineering Radio](http://www.se-radio.net)
 * [Podcast] - [The Changelog Podcast](https://changelog.com/podcast/)
 * [Podcast] - [The Silver Bullet Security Podcast with Gary McGraw](http://www.computer.org/web/computingnow/silverbullet)
-* [Podcast] - [Web Security Warriors](https://devchat.tv/web-security-warriors/)
-* [Blog] - [Infocodify/](http://www.infocodify.com/)
+* [Podcast] - [Web Security Warriors](https://devchat.tv/web-security-warriors)
```

Comment:
Sorry @vhf. I was just trying to help and now I believe I misunderstood you because I loaded that URL into my web browser and it loaded a webpage ok. That's all I meant by saying it wasn't dead yet. What episode info are you looking for? Do you know if that website is publishing new webcasts?